hugr-model/src/v0/text/print.rs

```diff
@@ -563,7 +572,7 @@ impl<'p, 'a: 'p> PrintContext<'p, 'a> {
                 outputs,
                 extensions,
             } => self.print_parens(|this| {
-                this.print_text("fn");
+                this.print_text("->");
```

Comment:
dumb question but should this come between inputs and outputs

Reply:
In s-expression based syntax it's common to have the function and operator names in the front, e.g. `(* 2 (+ 1 3))` or `(-> ins outs)`. So it's not a mistake. Overall the syntax needs some work anyway, but it won't be stabilised this month, so we can address syntax.